src/status_im2/contexts/wallet/events.cljs

```diff
@@ -122,28 +123,34 @@
               :on-success on-success
               :on-error   #(log/info "failed to derive address " %)}]]]})))
 
+(rf/reg-event-fx :wallet/add-account-success
+ (fn [{:keys [db]} [address]]
+   {:db (update db
+                :wallet              assoc
+                :navigate-to-account address
+                :new-account?        true)
+    :fx [[{:dispatch [:wallet/get-accounts]}]]}))
```

Comment:
Here we can omit the `{}` for dispatch:
`:fx [ [:dispatch [:wallet/get-accounts] ] ]`